sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -234,7 +234,9 @@ static const struct dmi_system_id sof_rt5682_quirk_table[] = {
 					SOF_SPEAKER_AMP_PRESENT |
 					SOF_MAX98360A_SPEAKER_AMP_PRESENT |
 					SOF_RT5682_SSP_AMP(0) |
-					SOF_RT5682_NUM_HDMIDEV(4)
+					SOF_RT5682_NUM_HDMIDEV(4) |
+					SOF_BT_OFFLOAD_SSP(1) |
+					SOF_SSP_BT_OFFLOAD_PRESENT
```

Comment:
This exact change is already in sof-dev?
c0e47db92682 ("ASoC: Intel: sof_rt5682: Update BT offload config for Rex")

Reply:
It is a different section, but I find these extremely confusing, almost identical commit messages doing different things.
Why not do it once?

Reply:
@ujfalusi - this was enabled recently, hence commit is done now

Reply:
The patch should be a fixup, imho:
`fixup! ASoC: Intel: sof_rt5682: Update BT offload config for Rex`
so we could merge them when sending upstream.

Reply:
Updated the commit message as suggested.